gstreamer-sdp/src/s_d_p_message.rs

```diff
@@ -165,167 +164,167 @@ impl SDPMessage {
         }
     }
 
-    pub fn get_attribute(&self, idx: u32) -> Option<SDPAttribute> {
+    pub fn get_attribute(&self, idx: u32) -> Option<&SDPAttribute> {
         unsafe {
-            from_glib_none(ffi::gst_sdp_message_get_attribute(self.to_glib_none().0, idx))
+            &*(from_glib_none(ffi::gst_sdp_message_get_attribute(self.to_glib_none().0, idx)) as *mut SDPAttribute)
```

Comment:
Just like above, store the pointer returned by the function, check for NULL and if so return None, otherwise the `&*` + cast